go.mod

```diff
@@ -1,12 +1,10 @@
 module go.mau.fi/whatsmeow
 
-go 1.23.0
-
-toolchain go1.24.0
+go 1.24
 
 require (
 	github.com/google/uuid v1.6.0
-	github.com/gorilla/websocket v1.5.0
+	github.com/gorilla/websocket v1.5.3
```

Comment:
The gorilla websocket package is not well maintained, they released a few bad versions, then a new version reverting most of the changes back. I'll just keep it on v1.5.0 until I get around to switching to https://github.com/coder/websocket